src/modules/rlm_rest/rlm_rest.c

```diff
@@ -119,6 +119,17 @@ static int rlm_rest_perform(rlm_rest_t *instance, rlm_rest_section_t *section, v
 	 *  HTTP body data into a single contiguous buffer.
 	 */
 	ret = rest_request_perform(instance, section, request, handle);
+
+	/*
+	 * Set an REST-HTTP-Code with the last code of http request.
+	 */
+	vp = fr_pair_afrom_num(request->packet, PW_REST_HTTP_CODE, 0);
+	if (vp) {
+		RDEBUG2("Adding REST-HTTP-Code = %d", rest_get_handle_code(handle));
+		vp->vp_integer = rest_get_handle_code(handle);
```

Comment:
I'm not sure how efficient the `rest_get_handle_code` method is, but it feels bad to call it twice. If you reverse the two lines above, the `RDEBUG` line could just get `vp->vp_integer` as the argument, instead of `rest_get_handle_code(handle)`.